schema/rss-service-plan-v4.json

```diff
@@ -467,9 +467,7 @@
       ],
       "properties": {
         "generation": {
-          "type": "integer",
-          "format": "uint64",
-          "minimum": 0.0
```

Comment:
Generations serialize like integers so I don't think this is an issue.

Reply:
Agreed, although I think our _deserialization_ of generations is slightly wrong - not relevant to this PR except that I went looking because of this comment! - so I filed #6865 (not at all urgent IMO).

Reply:
Added a comment linking to the issue.